src/content/docs/cheerpj3/11-guides/Networking.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Networking
+description: Networking with CheerpJ
+---
+
+CheerpJ 3.0 supports different networking scenarios that are divided into two categories:
+
+- Same-origin HTTP/HTTPS requests using fetch seamlessly.
+- Generalized networking via Tailscale.
+
+## Same origin HTTP/HTTPS requests
+
+A Java application running in the browser with CheerpJ can request resources to the local server (same origin) via fetch through the HTTP/HTTPS protocols. As expected, these requests are asynchronous. Remember that for a request to be considered same-origin, it needs to match the scheme, hostname and port between the requester and the receiver.
+
+The CheerpJ runtime API implements its own HTTP/HTTPS handler using the browser `fetch()`. To use this handler you need to pass into `cheerpjInit` with the `fetch` option as shown below:
```

Comment:
That is not actually required, we allowed fetch to be overridden for JNLP impl, but we should not document that.

By default the normal fetch is used anyway